lib/tpm_hash.c

```diff
@@ -97,3 +97,52 @@ int tpm_hash_compute_data(TSS2_SYS_CONTEXT *sapi_context, BYTE *buffer,
     free(bufferList);
     return rval == TPM_RC_SUCCESS ? 0 : -3;
 }
+
+//
+// This function does a hash on an array of data strings and re-uses syscontext
+//
+UINT32 tpm_hash_sequence( TSS2_SYS_CONTEXT *sysContext, TPMI_ALG_HASH hashAlg,
```

Comment:
sysContext --> sys_context or better yet sapi_context, hashAlg->hash_alg or hash_algorithm, and the same for the others.

Why is numBuffers UINT8 and not a size_t?